src/app.py

```diff
@@ -110,6 +109,119 @@ def update_figure(selected_drug, selected_years):
     dcc.Graph(id='demo_graph', figure=fig_demo)
 ])
 
+# Create Percentage of Overdoses Involving Opioids per Drug Type Chart
+# Create callback for the Percentage of Overdoses Involving Opioids per Drug Type Chart
+fig_percent_opioid_deaths = go.Figure()
+@app.callback(
+    [Output('percent-opioids', 'figure'),
```

Comment:
Consider to use underscore here to keep inline with other ids.
```suggestion
    [Output('percent_opioids', 'figure'),
```

Reply:
resolved.